context/src/test/java/io/grpc/ContextTest.java

```diff
@@ -759,7 +759,8 @@ public void initContextWithCustomClassLoaderWithCustomLogger() throws Exception
     StaticTestingClassLoader classLoader =
         new StaticTestingClassLoader(
             getClass().getClassLoader(),
-            Pattern.compile("(io\\.grpc\\.Context.*)|(io\\.grpc\\.ThreadLocalContextStorage.*)"));
+            Pattern.compile("(io\\.grpc\\.Context.*)|(io\\.grpc\\.ThreadLocalContextStorage.*)|"
+                + "(io\\.grpc\\.PersistentHashArrayMappedTrie)"));
```

Comment:
Why was this necessary? It actually looks partially broken (although in a way that probably doesn't cause any failure), since the nested classes wouldn't be copied.